packages/@aws-cdk/aws-s3/test/test.bucket.ts

```diff
@@ -201,6 +201,31 @@ export = {
     test.done();
   },
 
+  'bucket with block public access set to true'(test: Test) {
+    const stack = new cdk.Stack();
+    new s3.Bucket(stack, 'MyBucket', {
+      blockPublicAccess: true,
+    });
+
+    expect(stack).toMatch({
+      "Resources": {
+        "MyBucketF68F3FF0": {
+          "Type": "AWS::S3::Bucket",
+          "Properties": {
+            "PublicAccessBlockConfiguration": {
+              "BlockPublicAcls": true,
+              "BlockPublicPolicy": true,
+              "IgnorePublicAcls": true,
```

Comment:
Verify behavior for imported buckets

Reply:
What do you want me to check for imported buckets in the generated CF output? I don't get it.

Reply:
Specifically, I think the code not covered is `grantPublicAccess` should throw an error, no?